kubernetes-elasticsearch-cluster/es-master-stateful.yaml

```diff
@@ -13,13 +13,15 @@ spec:
       component: elasticsearch
       role: master
   serviceName: elasticsearch-master
-  replicas: 3
+  replicas: 1
```

Comment:
Can everything be reverted except for 23-24?

Reply:
Let's go back to:
replicas: 3
NUMBER_OF_MASTERS: 2

I've investigating an issue with baseline-baseline-explorer. It looks like an ES node went offline. So let's be resilient. (This is why I'm hesitent to make so many changes to pires, there may be unintended consequences.)

Reply:
Done.